### PR TITLE
Add lora adapter to helm deploy

### PR DESCRIPTION
**Pull request by BobClaerhout**

### CODE REVIEW

**Comment by sophokles73** [comment]
@BobClaerhout I think this needs to be rebased so that it doesn't contain all the cert changes again ...

**Comment by BobClaerhout** [comment]
@sophokles73, That's one possibility. Or, if we merge the other PR first, this PR will be updated and only have the second commit.

**Comment by BobClaerhout** [comment]
Rebased

File: deploy/src/main/deploy/lora-adapter.credentials [hunk 1]
```diff
@@ -0,0 +1,2 @@
+username=lora-adapter@HONO
+password=lora-secret
```
Comment (sophokles73) [inline deploy/src/main/deploy/lora-adapter.credentials]: the *lora-adapter@HONO* user also needs to be added to `deploy/example-permissions.json` in order for other services being able to authenticate the LoRa adapter using the Auth Server.

File: deploy/src/main/deploy/helm/templates/prometheus/prometheus-config.yaml [hunk 2]
```diff
@@ -24,6 +24,9 @@ data:
     {{- end }}
     {{- include "hono.prometheus.scrapeJob" (dict "dot" . "serviceName" "adapter-amqp-vertx") | nindent 4 }}
     {{- include "hono.prometheus.scrapeJob" (dict "dot" . "serviceName" "adapter-http-vertx") | nindent 4 }}
+    {{- if .Values.adapters.lora.enabled }}
+      {{- include "hono.prometheus.scrapeJob" (dict "dot" . "serviceName" "adapter-lora") | nindent 4 }}
```
Comment (sophokles73) [inline deploy/src/main/deploy/helm/templates/prometheus/prometheus-config.yaml]: the service names seems to be `adapter-lora-vertx`, doesn't it?

Comment (BobClaerhout) [reply deploy/src/main/deploy/helm/templates/prometheus/prometheus-config.yaml]: yes, you're right. Changed it

File: deploy/src/main/deploy/lora-adapter.credentials (hunk 1 above)
Comment (BobClaerhout) [reply deploy/src/main/deploy/lora-adapter.credentials]: Added

**Pull request merged**